tracer/src/Datadog.Trace/IAST/IastRequestContext.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="IastRequestContext.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.Iast;
+
+internal class IastRequestContext
+{
+    private VulnerabilityBatch? _vulnerabilityBatch;
+    private object _vulnerabilityLock = new();
+
+    public static void AddsIastTagsToSpan(Span span, IastRequestContext? iastRequestContext)
+    {
+        span.Tags.SetTag(Trace.Tags.IastEnabled, "1");
+
+        if (iastRequestContext != null)
+        {
+            iastRequestContext?.AddIastVulnerabilitiesToSpan(span);
+        }
```

Comment:
This is checking for null twice. Keep either the `if` or the `?`.
```suggestion
        iastRequestContext?.AddIastVulnerabilitiesToSpan(span);
```